machine_usage/views.py

```diff
@@ -288,16 +289,32 @@ def generate_failed_usage_form(request):
                 output[type_name] = []
             output[type_name].append(m.machine_name)
 
+        # for m in machines_idle:
+        #     type_name = m.machine_type.machine_type_name
+        #     if type_name not in output:
+        #         output[type_name] = []
+        #     output[type_name].append(m.machine_name)
+
         return render(request, 'machine_usage/forms/failed_usage.html', {"machines_in_use":output})
     else:
         machine_name = request.POST["machine_name"]
         machine = Machine.objects.get(machine_name=machine_name)
 
-        usage = machine.current_job
-        usage.clear_time = timezone.now()
-        usage.failed = True
-        usage.status_message = "Failed."
-        usage.save()
+        fail_log = {
+            "machine": machine_name,
+            "user": request.user.get_username(),
+            "filament": request.POST["filament_type"],
+            "percentage": request.POST["percentage"],
+            "error_msg": request.POST["error_msg"],
+            "observed_failure": request.POST.getlist("failure_type")
+        }
+
+        r = requests.post('https://t0tfudyw7g.execute-api.us-east-1.amazonaws.com/main', json = fail_log)
```

Comment:
Could you use a settings variable that is configured via [env variables like this one.](https://github.com/RPIForge/website/blob/ee92cbec37a005d7c640eb9e60735fb8155d1972/forge/settings.py#L147) That way we could update the endpoint at runtime